tests/x11/firefox/firefox_developertool.pm

```diff
@@ -9,6 +9,13 @@
 # without any warranty.
 
 # Summary: Case#1479522: Firefox: Web Developer Tools
+# - Launch xterm, kill firefox, cleanup previous firefox configuration, launch
+# firefox
```

Comment:
It would look nicer if you would add some indentation to continued lines - eg:

```
# - Launch xterm, kill firefox, cleanup previous firefox configuration, launch
#   firefox

```